tomster-player/app/templates/albums/album.hbs

```diff
@@ -5,9 +5,12 @@
   <div class="list-group-item">
     <div class="card">
       <div class="card-block">
+        {{#link-to 'albums.album.comment' model class='btn btn-primary'}}
+          New Comment
+        {{/link-to}}
```

Comment:
this actually needs to be in `albums/album/index.hbs` as the idea is to only show this on the `albums.album` route itself but not on the `albums.album.comment` route. By putting in into `albums/album/index.hbs` it will render into the outlet below on the `albums.album` route.